salesforce-gui/src/main/java/org/fundacionjala/salesforce/ui/skins/abstractPage/personalInformation/AbstractEditPersonalInformation.java

```diff
@@ -0,0 +1,140 @@
+package org.fundacionjala.salesforce.ui.skins.abstractPage.personalInformation;
+
+import org.fundacionjala.core.selenium.interaction.GuiInteractioner;
+import org.fundacionjala.salesforce.ui.commonPages.BasePage;
+import org.fundacionjala.salesforce.ui.skins.constants.personalInformation.Constants;
+import org.fundacionjala.salesforce.ui.entities.PersonalInformation;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.function.Supplier;
+
+public abstract class AbstractEditPersonalInformation extends BasePage {
```

Comment:
Please, when you have Pages, the class should end in "Page" to avoid confusions with "Form, Popup, or another component"

Reply:
Ok